neurolib/models/model.py

```diff
@@ -311,7 +299,7 @@ def clearModelState(self):
         self.state = dotdict({})
         self.outputs = dotdict({})
         # reinitialize bold model
-        if self.params.get("bold"):
+        if self.boldInitialized:
```

Comment:
Does this do the same thing as before? Reads like: if it is initialized, initialize it.

Reply:
The difference is that BOLD is now only initialized at the beginning of the first run. One only needs to clear the bold state with re-initialization if it has been initialized before.